src/constants/constantSettings.ts

```diff
@@ -4,4 +4,4 @@ export const TUTORIAL = 'tutorial.json'
 export const SCAM_WARNING = 'scamWarning.json'
 export const AAVE_WELCOME = 'aaveWelcome.json'
 export const TOKEN_TERMS_AGREEMENT = 'ttAgreement.json'
-export const STICKY_CONFIG = 'remoteConfigSticky.json'
+export const STICKY_CONFIG = 'remoteConfigSticky4.json'
```

Comment:
What's `remoteConfigSticky4.json`?

Reply:
Debug changes, shouldn't have made it in